src/components/dropdown/dropdown.stories.svelte

```diff
@@ -142,6 +142,7 @@
   <Link>Footer link</Link>
   <Link>Footer link</Link>
   <Dropdown value={undefined} {...args} mode="inline">
+    <Icon name="country-us" slot="left-icon" />
```

Comment:
huh interesting - I thought it wasn't kosher to slot a svelte element into a slot (and that you had to wrap it in a div).

Maybe they fixed it in Svelte 4?

Reply:
Maybe? I wasn't aware of it not being preferred.

Reply:
I think it was a compile error in Svelte 3